### DISCUSSION TITLE
es/posts/es-int/que-es-el-desarrollo-frontend

### DISCUSSION THREAD

**Discussion by giscus** [figueroaignacio/ai#2] [General]
# es/posts/es-int/que-es-el-desarrollo-frontend

Una guía introductoria al desarrollo frontend, explicando sus conceptos fundamentales, herramientas y prácticas clave para construir interfaces de usuario efectivas y atractivas. Aprende los fundamentos para empezar tu camino en el desarrollo web.

http://localhost:3000/es/posts/es-int/que-es-el-desarrollo-frontend

**Comment by figueroaignacio** [comment]
Hola mi nombre es Ignacio Figueroa y esto es un test de comentario.
